downstream/modules/platform/proc-settings-gw-additional-options.adoc

```diff
@@ -0,0 +1,32 @@
+[id="proc-settings-gw-other-options"]
+
+= Configuring additional platform options
+
+//Content divided into multiple procedures to address issue AAP-30592
+
+From the *Platform gateway settings* page, you can configure additional platform options.
+
+.Procedure
+. From the navigation panel, select {MenuSetGateway}.
+. The *Platform gateway settings* page is displayed. 
+. Click btn:[Edit platform gateway settings].
+. You can configure the following *Other settings*:
++
+* *JWT expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
++
+When authentication happens a JWT token is created for the user and that token is cached. 
+When subsequent calls happen to services such as automation controller or EDA, the token is taken from the cache and sent to the service. 
```

Comment:
Not sure if this is the first time you use EDA in this document, but note the guidance in the product names spreadsheet: https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039
Basically, they want us to avoid using the acronym, or only use it after the full name is written out first elsewhere in the doc.

Reply:
Good catch!